src/__tests__/views/loginView/__test__/loginView.test.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import { LoginView } from "../../../../views/loginView";
+
+const props = {
+  loginAction: jest.fn(),
+  onChangeRegister: jest.fn(),
+  history: {
+    push: val => jest.fn(val => val),
+  },
+};
+const initialState = {};
+const mockStore = configureStore();
+const store = mockStore(initialState);
+const user = {
+  username: "Roy",
+  email: "roy@andela.com",
+  password: "spidermansucks123",
+};
+const generateToken = (user) => {
+  const jwt = require("jsonwebtoken");
+  const token = jwt.sign(user, "sfhjjkdfjdkdfjdk", { expiresIn: 60 * 60 * 24 });
+  return token; 
+};
+const token = generateToken(user);
```

Comment:
Add a new line to make your work more readable

Reply:
I have added new lines